.github/workflows/build-glk.yml

```diff
@@ -0,0 +1,59 @@
+
+name: Protectli GLK
+on:
+  push:
+    branches:
+      - protectli_vault_glk/develop
+      - protectli_vault_glk/release
+  pull_request:
+    branches:
+      - protectli_vault_glk/develop
+      - protectli_vault_glk/release
+
+jobs:
+  build:
+    environment: Protectli
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - vendor: protectli
+            model: fw2b
+          - vendor: protectli
+            model: fw4b
+          - vendor: protectli
+            model: vp2410
+    steps:
+      - uses: actions/checkout@v3
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          # Checkout pull request HEAD commit instead of merge commit
+          # See: https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit
+          ref: ${{ github.event.pull_request.head.sha }}
+          # Fetch complete history
+          fetch-depth: 0
+      - name: Checkout all submodules
+        run: git submodule update --init --recursive --checkout
+      - name: Obtain Blobs 
```

Comment:
@Stojek139808 There is an ampty line at the end. Please fix your editor settings yo remove these automatically.